storage/storagepb/gen.go

```diff
@@ -12,6 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Package storagepb contains protobuf definitions and helpers used by various
+// storage implementations.
+//
+// TODO(pavelkalinnikov, v2): It's a bad practice to reuse the same protobuf
+// across multiple repositories/implementations. Remove it, and operate with
+// regular in-memory Golang structs through the storage API boundary.
```

Comment:
I don't think using protos as the definition of the data to be stored is necessarily a bad thing. There are clear guidelines for versioning protos and checking backwards compatibility. As it is "just" data, there are no methods to also worry about versioning on these structures either.

Having protos for transport between the storage and application layers doesn't force anyone to use them for storage. My advice would be that a storage layer which wants to store protos should define their own proto and copy between the formats. Is this what the comment means about reusing the protobuf in other repos?